src/server.lisp

```diff
@@ -261,7 +267,8 @@ DISPATCH-TABLE and LOGGING-STREAM are both required arguments.  TIMEOUT is of ty
                        (thread-count 5)
                        (logger (make-instance 'cl-syslog:rfc5424-logger
                                               :log-writer (cl-syslog:null-log-writer)))
-                       timeout)
+                       timeout
+                       debug)
```

Comment:
Can change this to `log-backtraces` or something more specific if `debug` feels to general.

Reply:
This feels OK to me.